test/openssl/test_pkcs7.rb

```diff
@@ -376,14 +378,9 @@ def test_split_content
 END
     pki_msg = OpenSSL::PKCS7.new(pki_message_pem)
     store = OpenSSL::X509::Store.new
-    pki_msg.verify(nil, store, nil, OpenSSL::PKCS7::NOVERIFY)
+    assert_equal(pki_msg.verify(nil, store, nil, OpenSSL::PKCS7::NOVERIFY), true)
```

Comment:
```suggestion
    assert_equal(true, pki_msg.verify(nil, store, nil, OpenSSL::PKCS7::NOVERIFY))
```

`test-unit`'s `assert_equal` takes the expected value as the first argument.

Reply:
Fixed!